framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1189,6 +1250,20 @@ private AnnotationMirror convertIntValToDoubleVal(AnnotationMirror intValAnno) {
     }
 
     /**
+     * Builds an ArrayLenRange annotation with {@code from} equal to its argument. The {@code to}
+     * field is set to Integer.MAX_VALUE, the default.
+     *
+     * @param val The value to use as the from in the ArrayLenRange annotation created. This value
+     *     must be non-negative.
+     */
+    public AnnotationMirror createMinLen(int val) {
```

Comment:
I find this method name confusing since it is actually creating an `@ArrayLenRange`.  This method is only called once, so could you just replace that call with one to `createArrayLenRangeAnnotation(val, MAX_VALUE)`